### PR TITLE
rtv: 1.4.2 -> 1.6.1

### PR DESCRIPTION
**Pull request by wedens**

### CODE REVIEW

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -13057,6 +13056,7 @@ let
     };
 
     propagatedBuildInputs = with self; [
+      requests
```
Comment (domenkozar) [inline pkgs/top-level/python-packages.nix]: probably `requests2`

Comment (wedens) [inline pkgs/top-level/python-packages.nix]: @domenkozar in [setup.py](https://github.com/michael-lazar/rtv/blob/master/setup.py#L6) is's just `requests` (I'm not familiar with python dependency management).

Comment (domenkozar) [inline pkgs/top-level/python-packages.nix]: It's a bit confusing, but `requests` means 1.x branch and `requests2` means 2.x branch

Comment (wedens) [inline pkgs/top-level/python-packages.nix]: ok. it works with `requests2`

File: pkgs/applications/misc/rtv/default.nix [hunk 2]
```diff
@@ -1,25 +1,28 @@
-{ stdenv, fetchFromGitHub, pkgs, python, pythonPackages }:
+{ stdenv, fetchFromGitHub, pkgs, lib, python, pythonPackages }:
 
 pythonPackages.buildPythonPackage rec {
-  version = "1.4.2";
+  version = "1.6.1";
   name = "rtv-${version}";
 
+  doCheck = false;
```
Comment (domenkozar) [inline pkgs/applications/misc/rtv/default.nix]: why did you disable tests? Add a note if it's not fixable and why

**Pull request merged**